config/plexora.txt

```diff
@@ -1,5 +1,5 @@
 # Uncomment this to enable Plexora. You probably won't need this, this is really only used by the main Monkestation server.
-#PLEXORA_ENABLED
+PLEXORA_ENABLED
```

Comment:
```suggestion
#PLEXORA_ENABLED
```

use `config/dev_overrides.txt` to change your config locally without pushing it to ur actual branch